compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -323,6 +323,10 @@
             { ParserRuleContext.END_OF_TYPE_DESC, ParserRuleContext.ARRAY_TYPE_DESCRIPTOR,
                     ParserRuleContext.OPTIONAL_TYPE_DESCRIPTOR, ParserRuleContext.PIPE };
 
+    private static final ParserRuleContext[] NEW_EXPR =
```

Comment:
better to rename NEW_EXPR  --> NEW_KEYWORD_RHS

Reply:
fixed in 401f175